pyproject.toml

```diff
@@ -26,7 +26,7 @@ classifiers = [
   "Operating System :: OS Independent",
 ]
 dependencies = [
-  "starlette",
+  "starlette[full]",
```

Comment:
The only problem I see with this is that it will make itsdangerous required for everyone and also include other dependencies from starlette[full] which is not ideal. I guess we can do two things:
- first make the SessionMiddleware import in authentication.py optional like this and raise an error if not installed in runtime:
```py
try:
    from starlette.middlewares import SessionMiddleware
Except ImportError:
    SessionMiddleware = None
```
- and as you suggested we can use starlette[full] but we add sqladmin[full] for this.